[2024-02-01 13:48:24,262] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: etl_postgresql.import_file_to_mysql manual__2024-02-01T13:48:23.614829+00:00 [queued]>
[2024-02-01 13:48:24,272] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: etl_postgresql.import_file_to_mysql manual__2024-02-01T13:48:23.614829+00:00 [queued]>
[2024-02-01 13:48:24,273] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2024-02-01 13:48:24,273] {taskinstance.py:1357} INFO - Starting attempt 1 of 2
[2024-02-01 13:48:24,274] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2024-02-01 13:48:24,301] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): import_file_to_mysql> on 2024-02-01 13:48:23.614829+00:00
[2024-02-01 13:48:24,307] {standard_task_runner.py:52} INFO - Started process 4022 to run task
[2024-02-01 13:48:24,310] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'etl_postgresql', 'import_file_to_mysql', 'manual__2024-02-01T13:48:23.614829+00:00', '--job-id', '271', '--raw', '--subdir', 'DAGS_FOLDER/etl_dag.py', '--cfg-path', '/tmp/tmpw39fbhfj', '--error-file', '/tmp/tmp8568zyh5']
[2024-02-01 13:48:24,312] {standard_task_runner.py:80} INFO - Job 271: Subtask import_file_to_mysql
[2024-02-01 13:48:24,389] {task_command.py:369} INFO - Running <TaskInstance: etl_postgresql.import_file_to_mysql manual__2024-02-01T13:48:23.614829+00:00 [running]> on host 0cb2a3b23a40
[2024-02-01 13:48:24,581] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=etl_postgresql
AIRFLOW_CTX_TASK_ID=import_file_to_mysql
AIRFLOW_CTX_EXECUTION_DATE=2024-02-01T13:48:23.614829+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-02-01T13:48:23.614829+00:00
[2024-02-01 13:48:27,945] {logging_mixin.py:115} INFO - Successfully install pandas
[2024-02-01 13:48:31,687] {logging_mixin.py:115} INFO - Successfully install sqlalchemy
[2024-02-01 13:48:34,570] {logging_mixin.py:115} INFO - Successfully install pymysql
[2024-02-01 13:48:34,693] {python.py:173} INFO - Done. Returned value was: import to mysql
[2024-02-01 13:48:34,712] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=etl_postgresql, task_id=import_file_to_mysql, execution_date=20240201T134823, start_date=20240201T134824, end_date=20240201T134834
[2024-02-01 13:48:34,766] {local_task_job.py:156} INFO - Task exited with return code 0
[2024-02-01 13:48:34,784] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
